package.json

```diff
@@ -71,9 +71,8 @@
     "@shopgate/eslint-config": "^1.0.4",
     "@shopgate/pwa-unit-test": "^1.3.0",
     "coveralls": "^3.0.0",
-    "eslint-import-resolver-babel-module": "4.0.0-beta.1",
-    "eslint-plugin-import": "^2.7.0",
```

Comment:
this dependency is still in the shrinkwrap. should it be removed from there or added here?